docs/reference/ml/ml-shared.asciidoc

```diff
@@ -843,6 +843,33 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-validation-loss-fold]
 For running jobs only, contains messages relating to the selection of a node to 
 run the job.
 
+//Begin data_counts
+`data_counts`:::
+(object)
+An object containing statistical data about the documents in the analysis.
++
+.Properties of `data_counts`
+[%collapsible%open]
+====
+`training_docs_count`:::
```

Comment:
Not a big deal, but in general we sort all these definition lists alphabetically and this section isn't sorted that way.